code/datums/status_effects/neutral.dm

```diff
@@ -43,5 +43,5 @@
 	duration = 40
 	alert_type = null
 
-/datum/status_effect/high_five/on_timeout()
-	owner.visible_message("[owner] was left hanging....")
+/datum/status_effect/high_five/on_remove()
```

Comment:
Having `on_timeout` is literally pointless, since it occurred just before `qdel(src)`, which leads to `on_remove` being called.